recipes/grpc/all/target_info/grpc_1.66.1.yml

```diff
@@ -0,0 +1,174 @@
+grpc_targets:
+    - name: "address_sorting"
+      lib: "address_sorting"
+    - name: "gpr"
+      lib: "gpr"
+      requires:
+        - abseil::absl_base
+        - abseil::absl_core_headers
+        - abseil::absl_log_severity
+        - abseil::absl_flags
+        - abseil::absl_flags_marshalling
+        - abseil::absl_any_invocable
+        - abseil::absl_check
+        - abseil::absl_log_globals
+        - abseil::absl_log
+        - abseil::absl_memory
+        - abseil::absl_random_random
+        - abseil::absl_status
+        - abseil::absl_cord
+        - abseil::absl_str_format
+        - abseil::absl_strings
+        - abseil::absl_synchronization
+        - abseil::absl_time
+        - abseil::absl_optional
+        - abseil::absl_variant
+    - name: "_grpc"
+      lib: "grpc"
+      requires:
+        - upb_json_lib
+        - upb_textformat_lib
+        - re2::re2
+        - zlib::zlib
+        - abseil::absl_algorithm_container
+        - abseil::absl_config
+        - abseil::absl_no_destructor
+        - abseil::absl_cleanup
+        - abseil::absl_flat_hash_map
+        - abseil::absl_flat_hash_set
+        - abseil::absl_inlined_vector
+        - abseil::absl_bind_front
+        - abseil::absl_function_ref
+        - abseil::absl_hash
+        - abseil::absl_type_traits
+        - abseil::absl_random_bit_gen_ref
+        - abseil::absl_random_distributions
+        - abseil::absl_statusor
+        - abseil::absl_span
+        - abseil::absl_utility
+        - c-ares::cares
+        - gpr
+        - openssl::ssl
+        - openssl::crypto
+        - address_sorting
+      frameworks: ['CoreFoundation']
+    - name: "grpc_unsecure"
+      lib: "grpc_unsecure"
+      requires:
+        - utf8_range_lib
+        - upb_message_lib
+        - zlib::zlib
+        - abseil::absl_algorithm_container
+        - abseil::absl_config
+        - abseil::absl_no_destructor
+        - abseil::absl_cleanup
+        - abseil::absl_flat_hash_map
+        - abseil::absl_flat_hash_set
+        - abseil::absl_inlined_vector
+        - abseil::absl_bind_front
+        - abseil::absl_function_ref
+        - abseil::absl_hash
+        - abseil::absl_type_traits
+        - abseil::absl_random_bit_gen_ref
+        - abseil::absl_random_distributions
+        - abseil::absl_statusor
+        - abseil::absl_span
+        - abseil::absl_utility
+        - c-ares::cares
+        - gpr
+        - address_sorting
+      frameworks: ['CoreFoundation']
+    - name: "upb_base_lib"
+      lib: "upb_base_lib"
+    - name: "upb_json_lib"
+      lib: "upb_json_lib"
+      requires:
+        - utf8_range_lib
+        - upb_message_lib
+    - name: "upb_mem_lib"
+      lib: "upb_mem_lib"
+    - name: "upb_message_lib"
+      lib: "upb_message_lib"
+      requires:
+        - upb_base_lib
+        - upb_mem_lib
+    - name: "upb_textformat_lib"
+      lib: "upb_textformat_lib"
+      requires:
+        - utf8_range_lib
+        - upb_message_lib
```

Comment:
This requirement is now outdated, https://github.com/grpc/grpc/blob/master/CMakeLists.txt#L3713C3-L3713C15 these two are replaced by the new components noted above.

This is also the case for some other targets that now depend on the new libraries